
const n1 = 10;
const n3 = 30;


console.log(n1);

try {
    console.log(n2);
} catch (error) {
    console.log(error);
}

console.log(n3);
